imod_coupler/drivers/ribametamod/ribametamod.py

```diff
@@ -259,6 +259,9 @@ def update(self) -> None:
             self.mf6.update()
 
         if self.has_ribasim:
+            # obtain modflow6 timestep, if not already done so
+            if self.delt is None:
```

Comment:
so the value is never updated by mf6 right? Since we use a default, it will never be None?